x-pack/plugins/security_solution/public/common/components/drag_and_drop/draggable_wrapper.tsx

```diff
@@ -126,24 +128,20 @@ const draggableContainsLinks = (draggableElement: HTMLDivElement | null) => {
   return links.length > 0;
 };
 
-const DraggableWrapperComponent: React.FC<Props> = ({
+const useHoverActions = ({
```

Comment:
Think this should be in it's own file?

Reply:
Yes you are right, I was just lazy on this. one. I will do it